### PR TITLE
EZP-30358: HTTP Cache purge when ContentService\{Hide,Reveal}ContentSignal is emitted

### PR DESCRIPTION
**Pull request by adamwojs**
> JIRA: https://jira.ez.no/browse/EZP-30358

## Description

Implementation of HTTP cache purge when `ContentService\HideContentSignal`/`ContentService\RevealContentSignal` is emitted.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.13.4@dev || ^7.3@dev",
+        "ezsystems/ezpublish-kernel": "^6.13.4@dev || ^7.5@dev",
```
Comment (mnocon) [inline composer.json]: Will it work for 2.4 branch? Ezplatform 2.4 uses both:
`"ezsystems/ezpublish-kernel": "~7.4.0@dev",`
`"ezsystems/ezplatform-http-cache": "~0.9.0@dev",`
(https://github.com/ezsystems/ezplatform/blob/2.4/composer.json#L36)
which will clash with this requirement if I'm not mistaken

Comment (lserwatka) [reply composer.json]: Good catch Marek. This code can not crash 2.4 version. Is this doable?

Comment (andrerom) [reply composer.json]: it will not afaik, if it does, then we also have same issue on 1.13

Comment (adamwojs) [reply composer.json]: Actually, it should work fine without `composer.json` modification (signal is not emitted => slot will not be executed). We will check this during QA.

Comment (andrerom) [reply composer.json]: Yeah, it's only solr having issues with this due to the use of abstract search slots, which is in retrospective not a good idea.

**Comment by lserwatka** [comment]
@micszo Great news, can we merge it?

**Comment by micszo** [comment]
@adamwojs did you want to fix CI before?

**Comment by andrerom** [comment]
Travis failure is basically same 2.4 and 1.13 will experience so indeed we need to get that green :)

**Comment by lserwatka** [comment]
Well done!

**Pull request merged**